Makefile

```
@@ -12,6 +16,7 @@ generate-integ-data:
 	@echo "Generating integration test data..."
 	.venv/bin/python tests/data_generator.py
 
-pr: lint test
+ci: lint test
+pr: lint format test
 
 .PHONY: *
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)